### DISCUSSION TITLE
Is it good to add more configuration options to the `evcxr.toml` file?

### DISCUSSION THREAD

**Discussion by baiguoname** [evcxr/evcxr#368] [Ideas]
This is an example of what the `evcxr.toml' file looks like:
![sna](https://github.com/evcxr/evcxr/assets/52092777/49a45b70-c503-46cc-b8a0-d730b2d0ef8f)
I can think of serveral advantages to this approache:
1. It's consistent with Rust syntax.
2. It's easy to parse the contents of the file to evcxr background sever.
3. if one day `Cargo.toml` file add more configuration options, we don't have to change the existing codes.
4. I can have more configuration options like `[source.crates-io]`.

**Comment by baiguoname** [comment]
@davidlattimore What do you think about it? If you think it's fine, I'm glad to make a pr.

**Comment by davidlattimore** [comment]
Sure, I think making more use of a declarative configuration file makes sense. I've occasionally regretted that I put so much configuration in place via commands rather than in a config file. A PR for that would be great. Thanks!
